src/app/images.tsx

```diff
@@ -108,7 +114,7 @@ export default function Images({
             <h2 className="font-bold pt-5" id="image_selection">
                 Image Selection
             </h2>
-            {image_error && <div>Please only enter valid image types: jpg, jpeg, cr2</div>}
+            {image_error && <div>Please only enter valid image types: jpg, jpeg, tif, tiff cr2</div>}
```

Comment:
This error message is still appearing for me when I enter directories, although it does let me add the directory